cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -579,10 +579,10 @@ std::string ThinClientPoolDM::selectEndpoint(
     getStats().setLocators((m_locHelper)->getCurLocatorsNum());
     getStats().incLoctorResposes();
 
-    char epNameStr[128] = {0};
-    std::snprintf(epNameStr, 128, "%s:%d", outEndpoint.getServerName().c_str(),
-                  outEndpoint.getPort());
-    LOGFINE("ThinClientPoolDM: Locator returned endpoint [%s]", epNameStr);
+    std::string epNameStr = outEndpoint.getServerName() + ":" +
+                            std::to_string(outEndpoint.getPort());
+    LOGFINE("ThinClientPoolDM: Locator returned endpoint [%s]",
```

Comment:
Log messages can be concatenated std::strings too. You can avoid the c-string format garbage.